docs/user/logs.md

```diff
@@ -415,11 +415,16 @@ The resulting OTLP record looks like the following example:
 
 #### Severity Parsing
 
-Typically, a log message has a log level written to a field `level`. Based on that, the agent tries to parse the log attribute `level` with a [severity parser](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/severity_parser.md). If that is successful, the log attribute is transformed into the OTel `severityText`and `severityNumber` attributes.
+Typically, a log message has a log level written to a field `level`. Based on that, the agent tries to parse the log attribute `level` with a [severity parser](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/severity_parser.md). If that is successful, the log attribute is transformed into the OTel attributes `severityText` and `severityNumber`.
```

Comment:
Please check the formatting. If I get it right, level, severityText, and severityNumber should be boldfaced.